test/models/participants_test.rb

```diff
@@ -1,77 +0,0 @@
-require "test_helper"
```

Comment:
Why was this removed?

Reply:
Old model that i used when first developing this feature.